docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -1,6 +1,6 @@
-=====================
-System access control
-=====================
+==============================
+Built-in system access control
```

Comment:
I went with "built-in" because this is how it is referred to in the first paragraph: "You can use one of the built-in implementations..." I definitely agree it *should* be a plural esp given that passage, but stuck with the singular previously in the title because I am not sure of the history of why it was titled that way.

Reply:
I had initially left it as-is, but felt there was still ambiguity in the search results.